src/graph/backend/dnnl/op_executable.cpp

```diff
@@ -1377,15 +1377,21 @@ softmax_executable_t::desc_t softmax_executable_t::create_desc(
     int64_t axis = op->get_attr<int64_t>(op_attr::axis);
     if (axis < 0) { axis += src.get_ndims(); }
 
-    const dnnl::algorithm algo
+    impl::alg_kind_t algo
             = op->get_kind() == dnnl_impl::op_kind::dnnl_logsoftmax
-            ? dnnl::algorithm::softmax_log
-            : dnnl::algorithm::softmax_accurate;
+            ? impl::alg_kind::softmax_log
+            : impl::alg_kind::softmax_accurate;
+
+    auto mode = op->get_attr<std::string>(op_attr::mode);
```

Comment:
for dnnl_logsoftmax, there's no attribute of mode?

Reply:
Thank you. I refactored the logic a bit here. Now we only get the attribute from dnnl_softmax op.